app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -134,6 +134,10 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector, HasSupp
                 .show()
     }
 
+    override fun invalidateToken(token: String) {
+        InstanceID.getInstance(this).deleteToken(token, GoogleCloudMessaging.INSTANCE_ID_SCOPE)
```

Comment:
If a user has more than one account, this would remove his token for other accounts also, dont?

Reply:
No, It will invalidate just that specific token.

Reply:
Actually, although the above is true, we're currently sharing the same token among different servers. I think when logging out and changing servers I think it will automatically refresh the token.